tensorflow/core/api_def/java_api/api_def_FilterDataset.pbtxt

```diff
@@ -0,0 +1,4 @@
+op {
```

Comment:
Instead of this, would it make sense to have the `OpGenerator` just skip operations with `func` attributes for now? That might be simpler than white-listing? Later on, when function valued attributes are supported, we can revisit?

Reply:
Well, looks like you saw right because a brand new op with a `func` attribute appeared when I last rebased, therefore was not skipped, and that broke the build. I agree to do a more robust solution for `func` attributes for now, I'll commit something.